### PR TITLE
[Feature1] - First Feature was implemented

### PR DESCRIPTION
**Pull request by udaypatel83**
Following changes were made;

New feature.txt file was added along with a.txt and Feature1 folder.

### CODE REVIEW

File: new-feature.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+feature1  branch file
```
Comment (patelu-qarik) [inline new-feature.txt:1]: good job.

Comment (patelu-qarik) [reply new-feature.txt:1]: Thanks

Comment (patelu-qarik) [reply new-feature.txt:1]: Thanks

Comment (udaypatel83) [reply new-feature.txt:1]: Thanks

**Review by patelu-qarik — APPROVED** [review]
good job. Approved

**Pull request merged**